dev/reference/mysql-compatibility.md

```diff
@@ -126,12 +130,13 @@ TiDB 支持 MySQL 5.7 中 **绝大多数的 SQL 模式**，以下几种模式除
 
 ### 默认设置的区别
 
-+ 默认字符集不同：
-    + TiDB 中为 `utf8`，相当于 MySQL 的 `utf8mb4`
-    + MySQL 5.7 中为 `latin1`，但在 MySQL 8.0 中修改为 `utf8mb4`
++ 默认字符集与 MySQL 不同：
+    + TiDB 中为 `utf8mb4`
+    + MySQL 5.7 中为 `latin1`，MySQL 8.0 中修改为 `utf8mb4`
 + 默认排序规则不同：
-    + MySQL 5.7 中使用 `latin1_swedish_ci`
-    + TiDB 使用 `binary`
+    + TiDB 中，`utf8mb4` 的默认排序规则为 `utf8mb4_bin`
+    + MySQL 5.7 中，`utf8mb4` 的默认排序规则为 `utf8mb4_general_ci`，MySQL 8.0 中修改为 `utf8mb4_0900_ai_ci`
+    + 请使用 [`SHOW CHARACTER SET`](/dev/reference/sql/statements/admin.md## `SHOW` 语句) 语句查看所有字符集的默认排序规则
```

Comment:
The link is incorrect here. @yikeke Please help to fix this in another PR. @bb7133 If you're not sure about the format, feel free to involve the I18N members to review it.

Reply:
Okay.